packages/rest/package-lock.json

```diff
@@ -4,6 +4,498 @@
+					"dev": true
+				}
+			}
+		},
+		"@istanbuljs/schema": {
+			"version": "0.1.2",
+			"resolved": "https://registry.npmjs.org/@istanbuljs/schema/-/schema-0.1.2.tgz",
+			"integrity": "sha512-tsAQNx32a8CoFhjhijUIhI4kccIAgmGhy8LZMZgGfmXcpMbPRUqn5LWmgRttILi6yeGmBJd2xsPkFMs0PzgPCw==",
+			"dev": true
+		},
+		"@loopback/build": {
+			"version": "3.1.1",
+			"resolved": "https://registry.npmjs.org/@loopback/build/-/build-3.1.1.tgz",
+			"integrity": "sha512-syl4CSKtkeCFhz2yLT4gl+nwvKgOLFagyx9sP8gTMov8t/DAKrcRxJin4pKWfz4hqpX/5sMrCFjmNS9LU1kOpA==",
+			"dev": true,
+			"requires": {
+				"@loopback/eslint-config": "^5.0.3",
+				"@types/mocha": "^7.0.1",
+				"@types/node": "^10.17.14",
+				"@typescript-eslint/eslint-plugin": "^2.19.0",
+				"@typescript-eslint/parser": "^2.19.0",
+				"cross-spawn": "^7.0.1",
+				"debug": "^4.1.1",
+				"eslint": "^6.8.0",
+				"eslint-config-prettier": "^6.10.0",
+				"eslint-plugin-eslint-plugin": "^2.2.1",
+				"eslint-plugin-mocha": "^6.2.2",
+				"fs-extra": "^8.1.0",
+				"glob": "^7.1.6",
+				"mocha": "^7.0.1",
+				"nyc": "^15.0.0",
+				"prettier": "^1.19.1",
+				"rimraf": "^3.0.1",
+				"source-map-support": "^0.5.16",
+				"typescript": "~3.7.5"
+			},
+			"dependencies": {
+				"cross-spawn": {
+					"version": "7.0.1",
+					"resolved": "https://registry.npmjs.org/cross-spawn/-/cross-spawn-7.0.1.tgz",
+					"integrity": "sha512-u7v4o84SwFpD32Z8IIcPZ6z1/ie24O6RU3RbtL5Y316l3KuHVPx9ItBgWQ6VlfAFnRnTtMUrsQ9MUUTuEZjogg==",
+					"dev": true,
+					"requires": {
+						"path-key": "^3.1.0",
+						"shebang-command": "^2.0.0",
+						"which": "^2.0.1"
+					}
+				},
+				"path-key": {
+					"version": "3.1.1",
+					"resolved": "https://registry.npmjs.org/path-key/-/path-key-3.1.1.tgz",
+					"integrity": "sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==",
+					"dev": true
+				},
+				"shebang-command": {
+					"version": "2.0.0",
+					"resolved": "https://registry.npmjs.org/shebang-command/-/shebang-command-2.0.0.tgz",
+					"integrity": "sha512-kHxr2zZpYtdmrN1qDjrrX/Z1rR1kG8Dx+gkpK1G4eXmvXswmcE1hTWBWYUzlraYw1/yZp6YuDY77YtvbN0dmDA==",
+					"dev": true,
+					"requires": {
+						"shebang-regex": "^3.0.0"
+					}
+				},
+				"shebang-regex": {
+					"version": "3.0.0",
+					"resolved": "https://registry.npmjs.org/shebang-regex/-/shebang-regex-3.0.0.tgz",
+					"integrity": "sha512-7++dFhtcx3353uBaq8DDR4NuxBetBzC7ZQOhmTQInHEd6bSrXdiEyzCvG07Z44UYdLShWUyXt5M/yhz8ekcb1A==",
+					"dev": true
+				},
+				"which": {
+					"version": "2.0.2",
+					"resolved": "https://registry.npmjs.org/which/-/which-2.0.2.tgz",
+					"integrity": "sha512-BLI3Tl1TW3Pvl70l3yq3Y64i+awpwXqsGBYWkkqMtnbXgrMD+yj7rhW0kuEDxzJaYXGjEW5ogapKNMEKNMjibA==",
+					"dev": true,
+					"requires": {
+						"isexe": "^2.0.0"
+					}
+				}
+			}
+		},
+		"@loopback/context": {
```

Comment:
Packages locks must not contains monorepo-local dependencies like `@loopback/context`, we have a CI job to detect that too - see the failed CI run for your PR here: https://travis-ci.com/strongloop/loopback-next/jobs/289645054#L236-L261

```
Invalid package-lock entries found!

  packages/repository/package-lock.json
    -> @loopback/build
    -> @loopback/context
    -> @loopback/core
    -> @loopback/eslint-config
    -> @loopback/metadata
    -> @loopback/testlab
  packages/rest/package-lock.json
    -> @loopback/build
    -> @loopback/context
    -> @loopback/core
    -> @loopback/eslint-config
    -> @loopback/http-server
    -> @loopback/metadata
    -> @loopback/openapi-spec-builder
    -> @loopback/openapi-v3
    -> @loopback/repository
    -> @loopback/repository-json-schema
    -> @loopback/testlab

Run the following command to fix the problems:

  $ npm run update-package-locks
```